plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -66,7 +68,9 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 
     setDefaultSourceLocator(launch, configuration);
 
-    if (ILaunchManager.DEBUG_MODE.equals(mode)) {
+    boolean debugMode = ILaunchManager.DEBUG_MODE.equals(mode);
```

Comment:
(I'd personally be curious to know what other modes people try. Such as "profile".)